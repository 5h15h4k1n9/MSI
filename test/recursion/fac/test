factorial(0)
factorial(5)
factorial(10)
exit
