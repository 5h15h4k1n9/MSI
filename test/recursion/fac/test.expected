(int) 1
(int) 120
(int) 3628800
Bye-bye!
